go/libkb/track.go

```diff
@@ -324,7 +324,7 @@ func (t TrackDiffNewEldest) GetTrackDiffType() keybase1.TrackDiffType {
 	return keybase1.TrackDiffType_NEW_ELDEST
 }
 func (t TrackDiffNewEldest) ToDisplayString() string {
-	return fmt.Sprintf("Changed from %s to %s", t.tracked, t.observed)
+	return fmt.Sprintf("Eldest key changed from %s to %s", t.tracked, t.observed)
```

Comment:
Maybe talk about "account reset" here instead? We don't really educate people on what an eldest key is.